### ISSUE TITLE
[bug] Error after logon-summary (and possibly other options): thread 'main' panicked at 

### ISSUE THREAD

**Issue by garyhooks** [Yamato-Security/hayabusa#1273]
Command:

> hayabusa-2.12.0-win-x64.exe logon-summary --directory "E:\winevt\Logs"

Error:
```
Loading detection rules. Please wait.

[00:00:13] 362 / 362   [====================] 100%

Scanning finished. Please wait while the results are being saved.
thread 'main' panicked at 
C:\Users\reguser\.cargo\registry\src\index.crates.io-6f17d22bba15001f\chrono-0.4.31\src\offset\local\mod.rs:176:46:
No such local time
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I did set the environment variable as suggested but this did not work

I also tried --output to send results to a CSV but no files were produced.

### Environment

Windows 10
Hayabusa version 2.12.0

**Comment by YamatoSecurity** [comment]
@garyhooks Thank you for reporting this. What is your timezone set to? and does hayabusa work if you add the `--UTC` option?

**Comment by garyhooks** [comment]
Hello,

Yes, --UTC did fix the issue and it now runs without any problem, thank you! 

The timezone just in case you needed it is UTC0 (London). 

Thanks,
Gary

**Comment by YamatoSecurity** [comment]
Thanks for the info. It seems to be a bug in chrono that was fixed in 0.4.32 (https://github.com/chronotope/chrono/releases)
We will ship new binaries with a new release that is coming out soon that should fix it. Please re-open this issue if the new version of Hayabusa (2.13.0) still panics when not using `--UTC`.
